### PR TITLE
fix(module:input): fix input disabled style bug

### LINKED ISSUE #103: [BUG] `nz-input` width `nzDisabled`
## I'm submitting a...

<pre><code>
[x] Bug report  
</code></pre>

## Current behavior

`nz-input` is ignored `.ant-input-disabled` class when set `[nzDisabled]="true"`

## Expected behavior

`.ant-input-disabled` should not be ignored.

```html
<nz-input [nzDisabled]="true" [nzSize]="'large'"></nz-input>
```

this is i get in [0.5.0-rc.2] version:

now

the text should be in disabled color i belive, just like:

aa

### PR DESCRIPTION
**Pull request by giscafer**
input component and directive

close #103

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Docs have been added / updated (for bug fixes / features)

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by vthinkxie** [comment]
please keep your commit history clean, you can use squash to remove unnecessary commit.

**Comment by giscafer** [comment]
@vthinkxie How can i remove the unnecessary commit in this pr？or i need to recreate a new pr？

i had try `git rebase -i xxx` to squash this two commit，it will be 3 commits in pr commit list。

**Comment by vthinkxie** [comment]
http://www.jianshu.com/p/964de879904a

**Comment by giscafer** [comment]
@vthinkxie it must use `git push -f` to force commit，i just use `git push`，thanks for the patient answer

File: src/components/input/nz-input.component.ts [hunk 1]
```diff
@@ -9,14 +9,14 @@ import {
   EventEmitter,
   ContentChild,
   forwardRef,
-  AfterContentInit, HostListener
+  AfterContentInit, HostListener, HostBinding
```
Comment (vthinkxie) [inline src/components/input/nz-input.component.ts]: please remove HostBinding as it was not used.

**Comment by giscafer** [comment]
@vthinkxie already remove the unused HostBinding

**Pull request merged**